http/jaxrs/src/test/java/io/quarkus/ts/security/core/BaseQuarkusBundleIT.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.ts.security.core;
+
+import static org.hamcrest.CoreMatchers.is;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class BaseQuarkusBundleIT {
+
+    @QuarkusApplication
+    static RestService app = new RestService();
```

Comment:
For these scenarios, you don't need to create the app. So, you can remove this declaration.
For rest assured, everything is configured for you using `RestAssured.given()`